lib/mastodon/media_cli.rb

```diff
@@ -183,6 +184,59 @@ def remove_orphans
 
       say("Removed #{removed} orphans (approx. #{number_to_human_size(reclaimed_bytes)})#{dry_run}", :green, true)
     end
+    # rubocop:enable Metrics/PerceivedComplexity
+
+    option :days, type: :numeric, default: 60, aliases: [:d]
+    option :skip_avatars, type: :boolean, default: false
+    option :include_follows, type: :boolean, default: false
+    option :concurrency, type: :numeric, default: 5, aliases: [:c]
+    option :verbose, type: :boolean, default: false, aliases: [:v]
+    option :dry_run, type: :boolean, default: false
+    desc 'remove-profile-media', 'Remove profile media files (headers, avatars)'
+    long_desc <<-DESC
+      Removes locally cached copies of profile headers and avatars. By default,
+      only accounts that are not followed by or following anyone locally are
+      pruned.
+      The --days option specifies how old the last webfinger request and update
+      to the user has to be before they are pruned. It defaults to 60 days.
+      If --skip_avatars is specified, avatars are not removed.
+      If --include_follows is specified, all non-local accounts will be pruned
+      irrespective of follow status.
+    DESC
+    def remove_profile_media
+      time_ago        = options[:days].days.ago
+      dry_run         = options[:dry_run] ? ' (DRY RUN)' : ''
+      include_follows = options[:include_follows]
+
+      purged_accounts = Concurrent::Set[]
+      processed, aggregate = parallelize_with_progress(
+        Account.where({ last_webfingered_at: Time.zone.at(0)..time_ago,
+          updated_at: Time.zone.at(0)..time_ago }).left_outer_joins(:user).where(users: { id: nil })
```

Comment:
I think you can use `..time_ago` now we've dropped Ruby 2.6 compat.

Also, why the join on `users` rather than using `Account.remote`?

Reply:
Done!